.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -68,7 +64,7 @@ on:
 jobs:
   build:
     name: Build + LIT
-    runs-on: ${{ inputs.build_runs_on }}
+    runs-on: [Linux, build]
```

Comment:
I think we can refactor this script to make it OS agnostic to re-use build and test workflow both on Linux and Windows.

Reply:
Maybe